crates/block-explorers/src/lib.rs

```diff
@@ -50,13 +50,36 @@ pub const ETHERSCAN_V2_API_BASE_URL: &str = "https://api.etherscan.io/v2/api";
 
 /// The Etherscan.io API version 1 - classic verifier, one API per chain, 2 - new multichain
 /// verifier
-#[derive(Clone, Default, Debug, PartialEq, Copy)]
+#[derive(Clone, Default, Debug, PartialEq, Copy, Eq, Deserialize, Serialize)]
+#[serde(rename_all = "lowercase")]
 pub enum EtherscanApiVersion {
-    #[default]
     V1,
+    #[default]
```

Comment:
@grandizzy can we make this default already?

Reply:
Another option would be to not specify a default here and specify a default on the forge library.